### ISSUE TITLE
Individual Initiative and contracting authority

### ISSUE THREAD

**Issue by SamirSaidani** [holacracyone/Holacracy-Constitution#435]
Through Individual Initiative, a Partner might contract for the Organization if all the conditions in 4.3 are met.
 
This is a problem I've met in real life: when a Partner is contracting for the Organization, the harm caused by this action is non reversible: the bullet is shot. This action might be done in good faith, but the partner might not be skilled enough to interpret a complex contract. In that case, 4.3 (c) "your action would not commit the Organization to any spending beyond what you’re already authorized to spend" might help, if the contract contains a financial risk, but I would prefer to be more explicit, because the Partner might miss the spot when reading the contract, or might underestimate the financial risk, or the contract might contain a legal risk threatening the organization. And even if we put some kind of domains or rules in the governance to mitigate the risk, the whole point of the individual initiative is precisely to be able to break these domains or policies if the Partner thinks it's the right thing to do.

I would suggest to add to 4.3:

(e) you are not contracting for the Organization unless you are authorized to do so.

Then we can put into the governance some rules defining the contracting authority, which can not be overridden by an individual initiative.

**Comment by brianjrobertson** [comment]
@SamirSaidani There would definitely be significant freedom/flexibility lost in preventing people from contracting via Individual Initiative (consider a "contract" doesn't just mean a typical legal document, but virtually any agreement with an outside party, even if just one made casually in an e-mail or even verbally).  So to consider this, I need more evidence from an actual case that illustrates why the current protections are insufficient to prevent harm.  Specifically, how is this a risk when §4.1.2 includes the rule "But if you believe your impact will be substantially difficult or expensive to undo, you need to get permission"?  Help me understand what's been tried and why current protections were insufficient in a real case study.

**Comment by SamirSaidani** [comment]
In my case, a Partner signed a contract with a great financial risk that might kill the company, and he believed in good faith (4.1.2) that the risk was zero, which turns out afterwards to be conflicting with the interpretation of "the legal stuff" Role. 

This action violates a domain (legal contract for the legal stuff Role) and a policy against signing this specific type of contract, but the Partner took the Individual Initiative pathway because of: 
- short timing (legal stuff Role didn't answer on time), 
- his interpretation that the contract won't cause harm by assessing the risk as zero,
- the great benefit on signing this big contract (overriding the policy against this type of contract).

Now the company can not undo the contract, since it has been signed by someone who did play by the game and has the authority to do so.

The company did put strong protections to prevent this situation (a domain and a policy), but Individual Initiative broke those firewalls.

I don't want to prevent people from contracting via Individual Initiative, I want to prevent people from overriding contracting authorization by playing the Individual Initiative card. In my case, the company has a policy like: don't sign a contract with potentially unlimited penalties (e.g high penalties for day of delay which might kill the company after x days). A Partner in good faith might (over)-estimate that he masters the delay, considering the contract is too good to let it go, and hence believe that his impact would not be expensive to undo since the risk from his point of view is zero.

My proposal "you are not contracting for the Organization unless you are authorized to do so" does not mean that people lose flexibility for contracting, but it means that the Organization can secure and finetune the contracting process as it is the case for the spending money process. Without this approach, my understanding is that by default a Partner is free to contract on behalf of the Organization to whatever contract he believes it's right. I think that the extent of this specific power is too dangerous, because those beliefs might be lethal for the Organization, and ultimately harmful for Partners. _We must give a chance to the Organization to question the beliefs of a Partner, especially when its own existence is jeopardized._

**Comment by brianjrobertson** [comment]
Thanks @SamirSaidani, the specific case & details are super helpful.  Before considering a constitutional change for something like this, I first need to understand why the situation isn't already covered by a relevant constitutional mechanism (and not just by a player's lack of understanding of the existing mechanisms, because if it's a case of lack of understanding, then training is the indicated path to address the issue).  In this case, it seems to me his action was a clear violation of multiple existing clauses in the constitution, so I'd first need to understand why I'm mistaken here and those existing clauses were insufficient.  Specifically:

1. Per §4.3.1(c), Individual Initiative is only allowed if it would not commit the Organization to any spending beyond what the actor is already authorized to spend.  In this case, if there is literally any chance (even 0.1%) of a delay, then signing that contract did potentially commit the organization to spending, and I doubt the argument that there is literally ZERO chance is a reasonable one.  (And even if the partner argued that was their interpretation in this case, a published Secretary interpretation should prevent such a liberal interpretation from ever being able to be used again by anyone.)
2. Per §4.1.2, you need permission before impacting a Circle-level domain if you believe the impact will be "substantially difficult or expensive to undo" - you shared why he might have believed it would not be expensive to undo, but it still sounds like it would be _**difficult**_ to undo given it's a locked-in contract, and note that clause uses an "or", so even that criteria alone is enough for that action to require explicit permission, even if it weren't "expensive" to undo.
3. It seems he may have violated §4.3.1(d) here as well, unless there was a reason he couldn't delay the initiative long enough to get permission to impact the domain without I-I?

**Comment by SamirSaidani** [comment]
I agree with you that §4.3.1(c) "Your action would not commit the Organization to any spending beyond what you’re already authorized to spend." is pretty clear and applies to my use case.

However I would like to make some observations:
- it's not clear how do you get authorization when there is a risk of spending (versus when you're about to spend money). 4.1.3 is all about spending money right away, but it's not clear on how to proceed when there is a (low) risk of spending money. For a high risk of spending, the partner might consider obvious to use the process, but when there is a very low risk of spending money from his point of view, the partner might consider this process not relevant ;
- in my use case, there was also buried in the (complex) contract a clause damaging the competitiveness of the organization (by preventing the org to apply to some tendering process for a while). The partner did not spot it, it requires some skills and patience to read a legal contract: so by signing it, he has damaged the org. 
In that case, §4.3.1(c) does not apply in my opinion. Although the concept of spending in 4.1.3 has a broader definition, it's not clear that spending is not only about money and the title 4.1.3 "Get Authorization Before Spending Money" adds to the confusion.

So my proposal is:
§4.3.1(c) - "You do not engage the Organization in any cost beyond what you’re already authorized to engage."
And generalize 4.3.1 by changing  "spending money" by "incurring costs".
4.1.3 - ""Get Authorization Before Incurring Costs" and change 4.1.3 accordingly, including the risk of incurring costs, money becoming a special case of a cost.

**Comment by SamirSaidani** [comment]
Oh, and about your point 2: "You believe that X is true" might be true even if X is false. Actually it doesn't matter if X is true or false, what matters is "do you believe it or not"... In my use case, this sentence was true. Belief has its share of irrational...

**Comment by matthiask** [comment]
I'm wondering a bit whether such a clause would necessarily have been seen at the right time in a conventionally structured organization. One way to protect the organization from such things is if no one can sign contracts alone; it's one of the reasons why we're only using joint signatures by two at our company.

**Comment by SamirSaidani** [comment]
> I'm wondering a bit whether such a clause would necessarily have been seen at the right time in a conventionally structured organization. One way to protect the organization from such things is if no one can sign contracts alone; it's one of the reasons why we're only using joint signatures by two at our company.

That's what we tried to organize with a "legal stuff" Role but Individual Initiative can override any policy under the right conditions, including joint signatures.
